woocommerce/woocommerce
# Storybook: Move code to `tools/`

### All Submissions:






### Changes proposed in this Pull Request:



Storybook contains stories relevant to packages as well as plugins, so a logical place to move its code is `tools/` as we prepare to integrate WooCommerce Admin client code into the core WooCommerce plugin.

This PR moves the code. In doing so, I have noticed two problems that currently exist in the implementation on `trunk` that should be addressed as follow ups.

* References to globals `moment` and `_wp_data` aren't found. Perhaps there is a regression because code introduced in https://github.com/woocommerce/woocommerce/pull/32396 would appear to address this problem.
* Certain WooCommerce styles don't appear where I would have expected them to. Some further digging will be required.
![image](https://user-images.githubusercontent.com/1922453/164564407-bb1a75b7-cc4b-4e00-a523-eb6f87c6578d.png)


Closes https://github.com/woocommerce/woocommerce/issues/32434

### How to test the changes in this Pull Request:

1. From project root, `pnpm install`
2. `pnpm run storybook`
3. Navigate to `http://localhost:6007/` and confirm everything loads correctly.
4. Note that the console reveals globals `moment` and `_wp_data` aren't found. This is to be addressed in a follow up

### Other information:




### FOR PR REVIEWER ONLY:
